README.md

```diff
@@ -11,6 +11,8 @@
 - When a user clicks the "Remove" button: The correct book is removed from the collection.
 - The correct book dissapears from the page.
 - The data is preserved in the browser's memory by using localStorage.
+- A Navigation bar is added with Three sections:Books list, Add book form and  Contact info.
+- When user  click on any of the links, the main area changes to display only one of the three content sections
```

Comment:
- [ ] (OPTIONAL) Kindly remove all the empty sections from your README file and instead add instructions on how a user can set up and run your project locally on their machine. This will make your README file more professional 💯 
- Your README:
![Screenshot 2022-06-29 193922](https://user-images.githubusercontent.com/85113567/176511837-838b9f05-2777-4174-936e-66f8057cf30a.png)